pylint/checkers/similar.py

```diff
@@ -889,8 +889,11 @@ def reduce_map_data(self, linter: PyLinter, data: list[list[LineSet]]) -> None:
         """Reduces and recombines data into a format that we can report on.
 
         The partner function of get_map_data()
+
+        Calls self.close() to actually calculate and report duplicate code.
         """
         Similar.combine_mapreduce_data(self, linesets_collection=data)
+        self.close()
```

Comment:
Simple as that is it! Great stuff!! :)

Reply:
Holy mother of Guido, was it *that* simple ?!